.github/buildomat/jobs/deploy.sh

```diff
@@ -239,13 +310,10 @@ do
 	retry=$((retry + 1))
 done
 
-
-# We also need to configure proxy arp for any services which use OPTE for external connectivity (e.g. Nexus)
-tar xf out/omicron-sled-agent.tar pkg/config-rss.toml
-SOFTNPU_MAC=$(dladm show-vnic sc0_1 -p -o macaddress | sed -E 's/[ :]/&0/g; s/0([^:]{2}(:|$))/\1/g')
-SERVICE_IP_POOL_START="$(sed -n 's/^first = "\(.*\)"/\1/p' pkg/config-rss.toml)"
-SERVICE_IP_POOL_END="$(sed -n 's/^last = "\(.*\)"/\1/p' pkg/config-rss.toml)"
-rm -r pkg
+# We also need to configure proxy arp for any services which use OPTE for
+# external connectivity (e.g. Nexus)
+SOFTNPU_MAC=$(dladm show-vnic sc0_1 -p -o macaddress \
+    | sed -E 's/[ :]/&0/g; s/0([^:]{2}(:|$))/\1/g')
```

Comment:
We can drop these (and subsequent) lines by just making sure `PXA_START/END` above also covers this range.

Reply:
I didn't do that because I'm also using `PXA_START` - `PXE_END` to configure the IP pool through the API in the end-to-end tests bootstrap. I'll adjust that and simplify this script a bit.